be/src/olap/txn_manager.cpp

```diff
@@ -172,7 +172,7 @@ void TxnManager::set_txn_related_delete_bitmap(
     std::unique_lock<std::mutex> txn_lock(_get_txn_lock(transaction_id));
     {
         // get tx
-        std::shared_lock rdlock(_get_txn_map_lock(transaction_id));
+        std::lock_guard<std::shared_mutex> wrlock(_get_txn_map_lock(transaction_id));
```

Comment:
map_lock is used to protect the txn map while txn lock is used to protect txn. It seems that, here we should acquire map read lock and released. then acquire txn read lock?